starter-code/src/functions-and-arrays.js

```diff
@@ -74,6 +186,24 @@ var wordsCount = [
   'disobedience',
   'matter'
 ];
+
+var searchResult = 0;
+
+var wordTried = "";
+
+function howManyTimes (wordsCount, wordTried){
+
+  for (i=0; i<wordsCount.length; i++){
+
+    if (wordTried === wordsCount[i]){ // returns true if the word we are looking is the only one on the array
+      searchResult += ;
+    }
+
+  }
+
+}
+
+
 // Bonus Quest
 
 var matrix = [
```

Comment:
Well done!
Your code is clean, you almost finished the full lab, which is encouraging! 
Keep up the good work :)